src/contracts/RelayOrderReactor.ts

```diff
@@ -0,0 +1,215 @@
+/* Autogenerated file. Do not edit manually. */
+/* tslint:disable */
+/* eslint-disable */
+import type {
+  BaseContract,
+  BigNumber,
+  BytesLike,
+  CallOverrides,
+  ContractTransaction,
+  PayableOverrides,
+  PopulatedTransaction,
+  Signer,
+  utils,
+} from "ethers";
+import type {
+  FunctionFragment,
+  Result,
+  EventFragment,
+} from "@ethersproject/abi";
+import type { Listener, Provider } from "@ethersproject/providers";
+import type {
+  TypedEventFilter,
+  TypedEvent,
+  TypedListener,
+  OnEvent,
+  PromiseOrValue,
+} from "./common";
+
+export type SignedOrderStruct = {
+  order: PromiseOrValue<BytesLike>;
+  sig: PromiseOrValue<BytesLike>;
+};
+
+export type SignedOrderStructOutput = [string, string] & {
+  order: string;
+  sig: string;
+};
+
+export interface RelayOrderReactorInterface extends utils.Interface {
```

Comment:
just noting out of date with latest main

Reply:
Ah yes will update